### ISSUE TITLE
Botocore dependency breaking Home Assistant on initial launch

### ISSUE THREAD

**Issue by epetousis** [andrewleech/ha_magiqtouch#4]
As mentioned in #1, there's an apparent issue with mandate's dependencies that seems to be causing an annoying error, requiring a restart or two either after initial installation of the component, or after updating Home Assistant. The error also blocks initial configuration, resulting in a "This integration does not support configuration via the UI" error. An example of the error is below:

```
2022-09-12 14:16:25.482 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration MagiQtouch: Unable to import component: cannot import name 'resolve_checksum_context' from 'botocore.client' (/usr/local/lib/python3.10/site-packages/botocore/client.py)
```

**Comment by andrewleech** [comment]
Yep these dependencies have been the no.1 issue I've run into while developing / installing the original version of this package. Some things required an older version of boto, others required a newer one, with conflicts all around.

I suspect the easiest path forward may be to get a working set of the required packages manually installed locally, then just copy them into this package in source so they don't need to be installed dynamically. There's plenty of other packages around that do this, even [`pip`](https://github.com/pypa/pip/tree/main/src/pip/_vendor)

**Comment by andrewleech** [comment]
This has hopefully been resolved in 069e56ba63770642ad8af449e1ed4a2ccc542cc8
